docs/flow/modin/core/dataframe/index.rst

```diff
@@ -0,0 +1,28 @@
+:orphan:
+
+Core Modin Dataframe Objects
+============================
+
+Modin paritions data to scale efficiently.
+To keep track of everything a few key classes are introduced: ``Dataframe``, ``Partition``, ``AxisPartiton`` and ``PartitionManager``.
+
+* `Dataframe` is the class conforming to Dataframe Algebra.
+* `Partition` is an element of a NxM grid which, when combined, represents the ``Dataframe``
+* `AxisPartition` is a joined group of ``Partition``-s along some axis (either rows or columns)
+* `PartitionManager` is the manager that implements the primitives used for Dataframe Algebra operations over ``Partition``-s
+
+Each :doc:`storage format </flow/modin/core/storage_formats/index>` may have its own implementations of these Core Dataframe's entities.
```

Comment:
Each execution engine and each execution system (storage format + execution engine) may have its own implementations of these Core Dataframe's entities too.

Reply:
added the info that you've provided

Reply:
but you removed info about storage format, why?

Reply:
did it accidentally, reverted it back